website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -1100,6 +1100,18 @@ tree, [2]
 Thus, `GroupByKey` represents a transform from a multimap (multiple keys to
 individual values) to a uni-map (unique keys to collections of values).
 
+Using <span class="language-java">`GroupByKey`</span> is straightforward:
```

Comment:
@denisecase I believe you need to wrap this whole sentence in the "language-java" class, so that it only appears if Java is the selected language.

Reply:
Thank you @pcoet - will do